E3db.podspec

```diff
@@ -9,7 +9,7 @@ E3DB provides a familiar JSON-based NoSQL-style API for reading, writing, and qu
                        DESC
 
   s.homepage         = 'https://tozny.com/'
-  s.license          = { :file => 'LICENSE.md' }
+  s.license          = { :type => 'TOZNY NON-COMMERCIAL LICENSE', :file => 'LICENSE.md' }
```

Comment:
Is this a requirement? Other repos where we listed "custom" or something for the license type broke once the repositories updated to require a proper SPDX slug for the type (which needed to become "proprietary"). Could that become a blocker with CocoaPods in the future?

Reply:
Doubt it would be a blocker, but do you think it's better just to use `Custom` here?